### ISSUE TITLE
How to run with single frame setting?

### ISSUE THREAD

**Issue by DeepRunner** [facebookresearch/VideoPose3D#20]
Hi, how to run the with single frame setting? Is the following commad correct?
python run.py -e 80 -k detectron_pt_coco -arc 1
Thanks!

**Comment by dariopavllo** [comment]
`-arc 1` results in a model that is too simple (only one input layer and one output layer). For the paper we used `-arc 1,1,1`, i.e. two residual blocks.

**Comment by DeepRunner** [comment]
Thanks!
Is the model similar to the ICCV17 simple baseline? The performance of the model improves about 10mm than them. Does this improvement come from the more accurate 2d detection?

**Comment by dariopavllo** [comment]
Yes, it becomes very similar. In that case, the improvement comes from multiple of factors, e.g. better 2D detections, hyperparameters, train/test augmentation, different training procedure (large batches, batchnorm momentum annealing, etc.)

**Comment by linjuxiaojiang** [comment]
What is your result(mpjpe) when you input 2d ground truth.

**Comment by linjuxiaojiang** [comment]
> Yes, it becomes very similar. In that case, the improvement comes from multiple of factors, e.g. better 2D detections, hyperparameters, train/test augmentation, different training procedure (large batches, batchnorm momentum annealing, etc.)



> Yes, it becomes very similar. In that case, the improvement comes from multiple of factors, e.g. better 2D detections, hyperparameters, train/test augmentation, different training procedure (large batches, batchnorm momentum annealing, etc.)

Hi
Can you specifically talk about the changes in the hyperparameters, train/test augmentation, different training procedure (large batches, batchnorm momentum annealing, etc.) of your network in the case of single frame input compared to 3d pose baseline.
Thanks!

**Comment by VRSEN** [comment]
@DeepRunner could you please share your single frame model if you still have it?
arseny9795@gmail.com
Thanks

**Comment by zzz0312** [comment]
> > 是的，它变得非常相似。在这种情况下，改进来自多个因素，例如更好的2D检测，超参数，训练/测试增强，不同的训练程序（大批量，批处理正态动量退火等）。
> 
> 嗨，您能具体谈谈与3d姿势基线相比，在单帧输入的情况下，您的网络的超参数，训练/测试增强，不同训练过程（大批量，批处理正态动量退火等）的变化吗？ 谢谢！

Hello, do you know the change of single frame compared with 3D baseline now?
